infrastructure/Taskfile.yml

```diff
@@ -1085,6 +1085,21 @@ tasks:
       preconditions:
       - *require_site
 
+    site:environment:sync:
+      desc: |
+        Synchonizes all file and database tables from environment
+        to another on make on environment a complete copy of the source.
+      deps: [lagoon:cli:config]
+      dir: "{{.dir_env}}"
+      vars:
+        SOURCE_PROJECT: "{{.SOURCE_PROJECT}}"
+        SOURCE_ENV: "{{.SOURCE_ENV}}"
+        TARGET_PROJECT: "{{.TARGET_PROJECT}}"
+        TARGET_ENV: "{{.TARGET_ENV}}"
+      cmds:
+        - lagoon ssh --project {{.TARGET_PROJECT}} --environment {{.TARGET_ENV}} -C "drush -y rsync @lagoon.{{.SOURCE_PROJECT}}-{{.SOURCE_ENV}}:%files @self:%files -- --omit-dir-times --no-perms --no-group --no-owner --no-times --chmod=ugo=rwX --delete; drush -y sql-sync @lagoon.{{.SOURCE_PROJECT}}-{{.SOURCE_ENV}} @self; exit;"
```

Comment:
:+1: I see that you added the `--delete` argument to `drush rsync`. Very nice!

With that in mind: Could you also add the `--create-db` option to `drush sql-sync`? This way we avoid any additional database tables added to the `moduletest` environment from lingering after a sync.

[We have discussed this in Zulip](https://reload.zulipchat.com/#narrow/channel/419623-DDF.2B/topic/Webmasterl.C3.B8sning/near/478069316).

Reply:
fixed